tests/jobs/scala/src/main/scala/TaskRunnerApp.scala

```diff
@@ -0,0 +1,33 @@
+import org.apache.spark.sql.SparkSession
+
+/**
+  * TaskRunnerApp creates a number of noop Spark tasks specified by 'numTasks' each of which
+  * sleeps for 'taskDurationSec'. The main goal of this class is to mimic long-running Spark
+  * applications (i.e. Spark Streaming) for the testing and troubleshooting purposes.
```

Comment:
LGTM. Only one nit: if the intention for this class is to generate "sleep tasks", should we rename it accordingly to be more precise?

Reply:
renamed to `MockTaskRunner` in order to avoid implementation exposure so it can be changed to something else instead of `Thread.sleep()`